index.bs

```diff
@@ -405,13 +411,11 @@ When this method is invoked, the user agent MUST execute the following algorithm
     an <a link-for='web'>opaque origin</a>, reject |promise| with a <a>DOMException</a> whose name is "NotAllowedError", and
     terminate this algorithm. Otherwise,
     - If the {{ScopedCredentialOptions/rpId}} member of {{options}} is not <a>present</a>, then set |rpId| to |callerOrigin|, 
-        and |rpIdHash| to the SHA-256 hash of |rpId|.
-    - If the {{ScopedCredentialOptions/rpId}} member of {{options}} is <a>present</a>, then invoke the procedure used for 
-        <a>relaxing the same-origin
-        restriction</a> by setting the `document.domain` attribute, using {{ScopedCredentialOptions/rpId}} as the given value
-        but without changing the current document's `domain`. If no errors are thrown, set |rpId| to the value of `host` as
-        computed by this procedure, and |rpIdHash| to the SHA-256 hash of the UTF-8 encoding of |rpId|. Otherwise, reject |promise| with a
-        <a>DOMException</a> whose name is "SecurityError", and terminate this algorithm.
+        and |rpIdHash| to the SHA-256 hash of the UTF-8 encoding of |rpId|.
+    - If the {{ScopedCredentialOptions/rpId}} member of {{options}} is <a>present</a>, then invoke the [[#algo-relax-same-origin-restriction]]
+        using these arugments: let |relaxingRpId| be {{ScopedCredentialOptions/rpId}}, and let |currentDocument| be the current <a>Document</a>. If no errors are thrown, set |rpId| to the value of `host` as
```

Comment:
Current document is not a well-defined term neither the W3C nor the WHATWG spec. One of the reason why issue [256](https://github.com/w3c/webauthn/issues/256) was raised was because the HTML5 spec definition doesn't specify which document object to interact with. 

[Active document](https://html.spec.whatwg.org/multipage/browsers.html#windows) is a possible replacement for "current document". However, to define specifically which document the origin is supposed to derive, you will need to also specify which browsing context you want. That may inevitably require us to rely on the caller vs. callee programming model ([271](https://github.com/w3c/webauthn/issues/271)) which is something we try to avoid.

One way to avoid this hassle would be to restrict the API to be only available on top level browsing context.